04-open.md

```diff
@@ -69,74 +70,132 @@ the process looks like this:
 
 This open model accelerates discovery:
 the more open work is,
-[the more widely it is cited and re-used](http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000308).
+[the more widely it is cited and re-used](http://dx.doi.org/10.1371/journal.pone.0000308).
 However,
 people who want to work this way need to make some decisions
 about what exactly "open" means in practice.
 
+## Version control as electronic notebook
+
+The benefits of version control in essence mean that, when used
+diligently, you can use version control as a form of electronic
+notebook for your computational work.
+
+* The conceptual stages of your work are documented, including who did
+  what and when. Every incremental step is stamped with an identifier
```

Comment:
I'd simplify this sentence to:

"Every step is stamped with an identifier, the commit-id, which is essentially unique for most purposes."